staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/types.go

```diff
@@ -95,6 +95,23 @@ type queue struct {
 	seatsInUse int
 }
 
+// queueWorkEstimate tracks the total work estimates comuted by taking
+// into account all requests in this queue.
+type queueWorkEstimate struct {
+	// InitialSeatsSum is the total number of initial seats
+	// associated with all requests in a given queue.
+	InitialSeatsSum int
+
+	// RequestedSeatsSum is the total number of requested seats
+	// associated with all requests in a given queue.
+	RequestedSeatsSum int
```

Comment:
"Requested" is not doing much useful work here.  We do _not_ have a distinction between, say, "requested" vs "granted" seats.

A more useful thing to mention in this field's name and comment is that we are talking about summing the _max_ seats.